zend_abstract_interface/functions/tests/php8/functions.cc

```diff
@@ -0,0 +1,381 @@
+extern "C" {
+#include "functions/functions.h"
+#include "zai_sapi/zai_sapi.h"
```

Comment:
These should use `#include <...>` right? I think that's what we decided?

Reply:
I think using `<>` makes since from the perspective of including these from the tracer since they live outside of the tracer project. But since these are tests for testing _within_ the ZAI project, `""` seems to make more sense to me.

In general, this feels most natural to me:
- `<>` = external project header
- `""` = internal project header

WDYT?